recipes/scikit-image/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ `uname` == "Darwin" ]
+then
+    export LDFLAGS="-L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
```

Comment:
Turns out Mac doesn't like `-rpath=`, but prefers `-rpath,` as [mentioned on SO]( http://stackoverflow.com/a/32400674 ). Seems to be working now that it is fixed.